tests/ReactTestApp/ios/ReactTestApp/AppDelegate.m

```diff
@@ -60,7 +60,10 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 - (NSURL *)sourceURLForBridge:(RCTBridge *)bridge
 {
 #if DEBUG
-  return [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"index" fallbackResource:nil];
+  // This is the original return from React Native
+  //return [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"index" fallbackResource:nil];
```

Comment:
Will it have any side effects that we check in this change?

Reply:
I checked it in for convenience and commented the old line out for future react native updates that may require updates to this file.
As for side effects, with this change, the metro bundler will return a larger amount of data in debug mode.  That could result in the loading of the js bundle to be a bit slower.  However, I have not noticed a big difference. I do not think it's that much of a performance hit.  

The Jenkins test step that consumes this is running in Release mode, so it will skip this line.  There will be no side-effects in the CI.